charts/sonarqube-dce/templates/ingress.yaml

```diff
@@ -40,30 +33,4 @@ spec:
         path: {{ .path}}
         pathType: {{ default "ImplementationSpecific" .pathType }}
   {{- end }}
-{{ else }}
-spec:
```

Comment:
We have quite some parameters that should be removed now. E.g., `.Values.ingress.annotations`, `.Values.ingress.tls`. Please update readme and values.yml

Reply:
I think there is a mistake here, 
 {{- if .Values.ingress.tls }}
  tls:
{{ toYaml .Values.ingress.tls | indent 4 }}
  {{- end -}}


Is supposed to stay as it is not part of the if/else

Also, .Values.ingress.annotations still exists in the chart